### PR TITLE
changes for 3.6.3. rc1

### PR DESCRIPTION
**Pull request by rdeutz**
Based on #15 changed files path and version tag

### CODE REVIEW

File: www/core/test/list_test.xml [hunk 1]
```diff
@@ -5,8 +5,8 @@
 	<extension name="Joomla" element="joomla" type="file" version="3.2.7" targetplatformversion="3.1" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.2" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.2.7" targetplatformversion="3.2" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.3" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.4" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.5" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml"/>
```
Comment (wilsonge) [inline www/core/test/list_test.xml]: These should still be 3.6.0 - because of the Joomla Update stuff

File: www/core/teststs/list_sts.xml [hunk 2]
```diff
@@ -4,8 +4,8 @@
 	<extension name="Joomla" element="joomla" type="file" version="3.2.7" targetplatformversion="3.1" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.2" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.2.7" targetplatformversion="3.2" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.3" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.4" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="3.6.0" targetplatformversion="3.5" detailsurl="https://update.joomla.org/core/teststs/extension_sts.xml" />
```
Comment (wilsonge) [inline www/core/teststs/list_sts.xml]: Same in this file

**Comment by rdeutz** [comment]
requested changed done @wilsonge

**Comment by wilsonge** [comment]
Nearly done. We need the update for the 3.6.2 to 3.6.3 release in as well. That's using the min dev version tag

**Comment by rdeutz** [comment]
file?

**Comment by wilsonge** [comment]
extension_sts.xml

File: www/core/teststs/extension_sts.xml [hunk 3]
```diff
@@ -57,4 +57,23 @@
 		<targetplatform name="joomla" version="3.6"/>
```
Comment (wilsonge) [inline www/core/teststs/extension_sts.xml]: so this needs a max_dev_level of 1 and a min_dev_level of 0

File: www/core/teststs/extension_sts.xml [hunk 4]
```diff
@@ -57,4 +57,23 @@
...
+		<downloads>
+			<downloadurl type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/3.6.3-rc1/Joomla_3.6.3-rc1-Release_Candidate-Update_Package.zip</downloadurl>
+		</downloads>
+		<tags>
+			<tag>stable</tag>
+		</tags>
+		<maintainer>Joomla! PLT</maintainer>
+		<maintainerurl>http://www.joomla.org</maintainerurl>
+		<section>STS</section>
+		<targetplatform name="joomla" version="3.6" max_dev_level="0"/>
```
Comment (wilsonge) [inline www/core/teststs/extension_sts.xml]: And this one should have a download url of Joomla_3.6.2_to_3.6.3-rc1-Release_Candidate-Patch_Package.zip and have a min_dev_level of 2

Comment (rdeutz) [inline www/core/teststs/extension_sts.xml]: ok, done by next push

File: www/core/teststs/extension_sts.xml (hunk 3 above)
Comment (rdeutz) [inline www/core/teststs/extension_sts.xml]: ok, done by next push

File: www/core/teststs/extension_sts.xml [hunk 5]
```diff
@@ -65,15 +65,15 @@
...
+			<downloadurl type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/3.6.3-rc1/Joomla_3.6.2_to_3.6.3-rc1-Release_Candidate-Patch_Package.zip</downloadurl>
 		</downloads>
 		<tags>
 			<tag>stable</tag>
 		</tags>
 		<maintainer>Joomla! PLT</maintainer>
 		<maintainerurl>http://www.joomla.org</maintainerurl>
 		<section>STS</section>
-		<targetplatform name="joomla" version="3.6" max_dev_level="0"/>
+		<targetplatform name="joomla" version="3.6" max_dev_level="2"/>
```
Comment (wilsonge) [inline www/core/teststs/extension_sts.xml]: this should be min_dev_level  I think?

Comment (rdeutz) [inline www/core/teststs/extension_sts.xml]: I hate it when I do something a hardly understand :-)

**Comment by zero-24** [comment]
There is no reason to lock to 3.6.0? All versions (expected 2.5) have a one step update? Or do i miss something?

**Comment by wilsonge** [comment]
Did I merge that one? I'm sorry I'm knackered today..... :(

**Comment by zero-24** [comment]
Yes? https://github.com/joomla/joomla-cms/pull/11689

**Comment by wilsonge** [comment]
*facepalm*

fixed. does this look better now?

**Comment by zero-24** [comment]
I have fixed it. Without my changes 3.6.3-rc1 could only be download by 3.6 Installes now everyone can :smile:

**Comment by zero-24** [comment]
Is there no release announcment planed? If yes than add the URL where i have added the Github Release tag :smile:

**Comment by wilsonge** [comment]
No - we don't usually for patch releases

**Comment by wilsonge** [comment]
I'll push this in my lunch break in 30 minutes

**Comment by zero-24** [comment]
Thanks! Please don't forget the nightly xmls ;) :+1:

**Comment by rdeutz** [comment]
changes are pushed to the update-server

**Pull request merged**